### PR TITLE
fix more bugs

### PR DESCRIPTION
**Pull request by GraceGardner**
What (if any) features are you implementing?
Bug fixes:
- If player was deleted, and user went back home the deleted player was still on the leader board until page was refreshed. 
- When user pressed OK from error modal, user was shown a white screen. 

What (if anything) did you refactor?
- Error Modal
- Verification Modal 

Were there any issues that arose?
N/A

Is there anything that you need from your teammate?
Review Please! :) 

Any other comments, questions, or concerns?
N/A

### CODE REVIEW

File: src/Components/ErrorModal.js [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
+import { useNavigate } from 'react-router-dom'
 import { Errors } from '../Contexts/ErrorsProvider'
 import '../Styles/ErrorModal.scss'
 
 const ErrorModal = () => {
 
+  const navigate = useNavigate()
   const {errorMessage, setErrorMessage} = useContext(Errors)
 
+  const errorRedirect = () => {
```
Comment (oacortez) [inline src/Components/ErrorModal.js:11]: Good function! This fixed our bug to redirect users back home

File: src/Components/VerifyDeletionModal.js [hunk 2]
```diff
@@ -2,16 +2,21 @@ import React, { useContext } from 'react'
...
 const VerifyDeletionModal = ({name, id, toggleVerify}) => {
 
+  const { getLeagueMembers } = useContext(LeagueMembers)
   const { setErrorMessage } = useContext(Errors)
   const navigate = useNavigate()
 
   const removePlayer = () => {
     deletePlayer(id)
-    .then (() => navigate('/'))
+    .then (() => {
```
Comment (oacortez) [inline src/Components/VerifyDeletionModal.js:16]: Thanks for implementing our getLeagueMembers function to updated our deleted users when going back to home view

**Review by oacortez — APPROVED** [review]

File: src/Components/ErrorModal.js [hunk 3]
```diff
@@ -1,21 +1,28 @@
 import React, { useContext } from 'react'
+import { useNavigate } from 'react-router-dom'
 import { Errors } from '../Contexts/ErrorsProvider'
 import '../Styles/ErrorModal.scss'
 
 const ErrorModal = () => {
 
+  const navigate = useNavigate()
```
Comment (philalewis) [inline src/Components/ErrorModal.js:8]: Good catch!

File: src/Components/VerifyDeletionModal.js [hunk 4]
```diff
@@ -2,16 +2,21 @@ import React, { useContext } from 'react'
 import { deletePlayer } from '../apiCalls'
 import { useNavigate } from 'react-router-dom'
 import { Errors } from '../Contexts/ErrorsProvider'
+import { LeagueMembers } from '../Contexts/LeagueMembersProvider'
 import '../Styles/VerifyDeletionModal.scss'
 
 const VerifyDeletionModal = ({name, id, toggleVerify}) => {
 
+  const { getLeagueMembers } = useContext(LeagueMembers)
```
Comment (philalewis) [inline src/Components/VerifyDeletionModal.js:10]: Good catch here, too!

**Review by philalewis — APPROVED** [review]

**Pull request merged**